setup/borg/cli.md

```diff
@@ -6,111 +6,130 @@ parent: Borg
 grand_parent: Setup
 description: "Instructions on how to set up Borg Backup from the command line on Linux and macOS."
 ---
+
 # How to Set Up Borg Backups from the Command Line on Linux and macOS
 {: .no_toc }
 
+Reviewed in August 2023
+
 1. TOC
 {:toc}
 
 {: .note }
 You should be comfortable using the command line. If you prefer a graphical, client, look into our [Vorta Desktop Client](/setup/vorta) instead. These instructions should work on macOS and popular Linux flavors, like Debian, Ubuntu, as well as Red Hat, Fedora and CentOS.
 
 {: .note }
-Borg will get a [major upgrade](https://borgbackup.readthedocs.io/en/2.0.0b3/changes.html#change-log-2-x) to version 2 later this year, which brings many improvements and removes legacy code- and features. Most commands stay the same. Where there is a difference, you can choose the right version using the tabs above the code snippet. *BorgBase* itself supports all the latest release of every major branch and you can choose the preferred version for each repository.
```

Comment:
Since you add many formatting changes, please add a link to the used markdown styleguide, so we can refer to it in the future in the README.